pkg/keccak/contrib/Modes/keccak1600.h

```diff
@@ -0,0 +1,211 @@
+/*
+
+Implementation by the Keccak, Keyak and Ketje Teams, namely, Guido Bertoni,
```

Comment:
I have one question regarding the copyright: from where does this file comes from ? Is it an original contribution on top of the package API ?
If yes, the header (copyright, license, authoship, doxygen) should be adapted following RIOT's convention and https://github.com/RIOT-OS/RIOT/pull/7903/files#r192867793 should also be addressed (don't use define but a typedef as requested @kaspar030)

Reply:
I'm not going to lie to you, it's been so long that I don't really remember well. I *think* with some confidence but no certainty that it's a modified version of a file from the original package. I don't think I would have included the copyright notice otherwise.
I didn't forget about @kaspar030 's request but there was still an open question that I'd liked answered.

Reply:
So, assuming that my memory serves me right, are we going to leave the copyright as it is or change it anyway?